sp800-63a/sec5_proofing.md

```diff
@@ -36,10 +36,10 @@ This section provides quality requirements for identity evidence collected durin
+|Fair|- The issuing source of the evidence confirmed the claimed identity through an identity proofing process.<br><br>- The issuing process for the evidence means that it can reasonably be assumed to have been delivered into the possession of the person to whom it relates.<br><br>- The evidence:<br>&nbsp;&nbsp;&nbsp;&nbsp;- contains at least one reference number that uniquely identifies the person to whom it relates. <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- contains a photograph or biometric template (any modality) of the person to whom it relates.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- can have ownership confirmed through KBV.<br><br>- Where the evidence includes digital information, that information is protected using cryptographic or proprietary methods, or both, and those methods ensure the integrity of the information and enable the authenticity of the claimed issuing source to be confirmed. <br><br>- Where the evidence includes physical security features, it requires proprietary knowledge to be able to reproduce it.<br><br> - The issued evidence is unexpired.|
+|Strong|- The issuing source of the evidence confirmed the claimed identity through written procedures designed to enable it to form a reasonable belief that it knows the true identity of the person. Such procedures shall be subject to recurring oversight by regulatory or publicly accountable institutions. For example, the Customer Identification Program guidelines established in response to the USA PATRIOT Act of 2001 or the [Red Flags Rule](#rfr), under Section 114 of the Fair and Accurate Credit Transaction Act of 2003 (FACT Act).<br><br>- The issuing process for the evidence ensured that it was delivered into the possession of the subject to whom it relates.<br><br>  - The issued evidence contains at least one reference number that uniquely identifies the person to whom it relates.<br><br>- The full name on the issued evidence must be the name that the person was officially known at the time of issuance. Pseudonyms, aliases, and initials for first given name and surname are not permitted.<br><br>- The issued evidence contains a photograph or biometric template (any modality) of the person to whom it relates.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>- The applicant proves possession of an AAL2 authenticator bound to an IAL2 identity, at a minimum.<br><br>- Where the issued evidence includes digital information, that information is protected using cryptographic or proprietary methods, or both, and those methods ensure the integrity of the information and enable the authenticity of the claimed issuing source to be confirmed.<br><br>  - Where the issued evidence contains physical security features, it requires proprietary knowledge and proprietary technologies to be able to reproduce it.<br><br>- The evidence is unexpired.|
```

Comment:
Must it be the first name that is included? How about one given name, e.g., J. Edgar Hoover? (same comment for Superior)